src/System.Azure.Experimental/System/Azure/StorageAccessSignature.cs

```diff
@@ -11,6 +12,75 @@ namespace System.Azure.Authentication
 {
     public static class StorageAccessSignature
     {
+        public static bool TryWrite2(Span<byte> output, Sha256 hash, string verb, string canonicalizedResource, DateTime utc, out int bytesWritten)
```

Comment:
Rename TryWrite2 to something more meaningful.

Reply:
The Idea was that I will implement it using SpanWriter, but I discovered SpanWriter does not work for this routine. I will remove it for now, fix SpanWriter, and then implement this routine again.